dbt/models/marts/users/dim_user_course_activity.sql

```diff
@@ -70,12 +70,10 @@ combined as (
 	join course_structure cs
 		on ul.script_id = cs.script_id 
         and ul.level_id = cs.level_id 
-    join users u
-        on ul.user_id = u.user_id
 	join school_years sy 
 		on ul.created_at 
             between sy.started_at and sy.ended_at
-    {{ dbt_utils.group_by(4) }}
+    {{ dbt_utils.group_by(3) }}
```

Comment:
Adding back the user_type, the group would be four fields
```suggestion
    {{ dbt_utils.group_by(4) }}
```

Reply:
won't user_type be 'student' every time @nataliazm99?